desktop/apps/consignments2/stylesheets/landing/desktop.styl

```diff
@@ -65,6 +62,10 @@
     &__headline
       margin-bottom 38px
       garamond-size('xl-headline')
+
+      @media screen and (max-width: 768px)
```

Comment:
I don't believe we have these media queries codified anywhere... and we probably should put something [in here](https://github.com/artsy/force/tree/master/desktop/components/stylus_lib) 😁